src/fabric/src/fabric2_fdb.erl

```diff
@@ -487,15 +488,27 @@ get_doc_body_wait(#{} = Db0, DocId, RevInfo, Future) ->
     fdb_to_doc(Db, DocId, RevPos, [Rev | RevPath], BodyRows).
 
 
-get_local_doc(#{} = Db0, <<?LOCAL_DOC_PREFIX, _/binary>> = DocId) ->
+get_local_doc(#{} = Db, <<?LOCAL_DOC_PREFIX, _/binary>> = DocId) ->
```

Comment:
What if instead of exposing `fdb_to_local_doc` we instead export a new `get_local_doc(Db, DocId, Binary)` function that delegates to `fdb_to_local_doc`? Its obviously not a huge difference but the `fdb_to_X` functions were until this point internal only helpers and it feels odd to expose them.

Also, removing the `Db` argument, even though its unused changes the `fdb_to_X` pattern and also prevents us from easily extending the implementation in ways that may require it. For instance, if we start allowing `_local` documents to be larger than 100k we may need to look up the rest of the body or some such.

Reply:
Good point. Agree on adding Db arg back